docs/sections/contributor_guide/index.rst

```diff
@@ -7,3 +7,4 @@ Contributor Guide
    developer_setup
    code_quality
    fork_pr_model
+   documentation
```

Comment:
There's nothing in `documentation.rst` yet, but adding it to the table of contents eliminated a warning message from `make html`.